scripts/run_tests.sh

```
@@ -0,0 +1,15 @@
+!#/usr/bin/env bash
```

Shellcheck

[error] Use #!, not !#, for the shebang.

(SC1084)